spec/support/with_nomis_alerts_seed_data.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 RSpec.shared_context 'with Nomis alerts reference data' do
+  let(:serco_supplier) { create :supplier, key: 'serco', name: 'Serco' }
```

Comment:
Doesn't look like this gets called anywhere so is probably redundant.

Reply:
We're using it in the api stories

Reply:
this is called in 4 places.

Reply:
Sorry, I didn't spot it was a shared context - my mistake.